pkg/cmd/config-certificates.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2025 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8c.io/kubeone/pkg/certificate"
+)
+
+func configCertificatesCmd(rootFlags *pflag.FlagSet) *cobra.Command {
```

Comment:
`config` seems like a bad root command for this command. To me, `config` has always been about manipulating the KubeOne configuration (`kubeone.yaml`) and other relative configurations (MachineDeployments, list of images...).

What do you think about having this as a dedicated root command, e.g. `kubeone certs renew`? Or maybe even `kubeone apply renew-certs`?